acceptance/bundle/variables/prepend-workspace-var/script

```diff
@@ -0,0 +1,2 @@
+echo /Workspace should be prepended on all paths, but it is not the case:
+$CLI bundle validate -o json
```

Comment:
Please include a link to this PR and #2181 as well. Need `git log` otherwise to figure out where this comes from.

Reply:
I've added link to #2181.